articles/openshift/tutorial-create-cluster.md

```diff
@@ -29,21 +29,6 @@ To create an Azure Red Hat OpenShift cluster, verify the following permissions o
 |**User Access Administrator**|X|X| |
 |**Contributor**|X|X|X|
 
-### Install the `az aro` extension
-The `az aro` extension allows you to create, access, and delete Azure Red Hat OpenShift clusters directly from the command line using the Azure CLI.
-
-Run the following command to install the `az aro` extension.
-
-```azurecli-interactive
-az extension add -n aro --index https://az.aroapp.io/stable
-```
-
-If you already have the extension installed, you can update by running the following command.
-
-```azurecli-interactive
-az extension update -n aro --index https://az.aroapp.io/stable
-```
-
 ### Register the resource provider
```

Comment:
@mjudeikis Do we still need this entire section on registering the resource provider? Or maybe just keep the first step?

Reply:
We still need it.

Reply:
@lamek We should remove the section on verifying that the extension is registered in the section below.

Reply:
We still need it.